compose-shell-param.js

```diff
@@ -137,7 +138,7 @@ xtag.register('compose-shell-param', {
   },
 
   methods: {
-    toggle: function(){ this.visible = !this.visible },
+    toggle: function(){ this.add = true; this.visible = !this.visible },
```

Comment:
Can we move that "add" logic elsewhere? It could be the responsibility of the shell to decide to focus it when the button is clicked. Here maybe: https://github.com/compose-ui/shell/blob/master/compose-shell.js#L49